pkg/scheduler/cache/usagedb/api/interface.go

```diff
@@ -37,6 +37,9 @@ type UsageParams struct {
 	WindowSize *time.Duration `yaml:"windowSize" json:"windowSize"`
 	// Window type for time-series aggregation. If not set, defaults to sliding.
 	WindowType *WindowType `yaml:"windowType" json:"windowType"`
+	// Cron string for the usage. If not set, the usage will not be aggregated.
+	CronString string `yaml:"cronString" json:"cronString"`
```

Comment:
"CronString" is the "type", not the "what" this param does.. Needs a better name, maybe "ResetTime", "ResetCron", "TumblingWindowCron" etc.
Also, the docstring doesn't make sense at all.

Reply:
Renamed and changed the description